src/client/loadLocale.js

```diff
@@ -1,55 +1,28 @@
-import config from '../../config';
-import { repeat, partialRight } from 'lodash';
-import { t } from 'tcomb-react';
-
-function longerStrings(data) {
-  if (!t.Num.is(config.localStringLengthMultiplier) || config.localStringLengthMultiplier === 1) {
-    return data;
-  }
-
-  const _repeat = partialRight(repeat, config.localStringLengthMultiplier);
-  const _longerStrings = msgs => Object.keys(msgs).reduce((ac, k) => ({
-    ...ac,
-    [k]: t.Str.is(msgs[k]) ? _repeat(msgs[k]) : _longerStrings(msgs[k])
-  }), {});
-
-  return {
-    ...data,
-    messages: _longerStrings(data.messages)
-  };
-}
-
-function loadMessagesFor(locale) {
-  return new Promise(resolve => {
-    require.ensure([], require => {
-      resolve(
-        longerStrings(require('../app/locales/' + locale)) // eslint-disable-line prefer-template
-      );
-    });
-  });
-}
-
-function hasLocaleData(locale) {
-  // http://formatjs.io/guides/runtime-environments/#polyfill-node
-  return Intl.NumberFormat.supportedLocalesOf(locale)[0] === locale &&
-    Intl.DateTimeFormat.supportedLocalesOf(locale)[0] === locale;
-}
-
-function hasIntlFor(locale) {
-  return window.Intl && hasLocaleData(locale);
-}
-
-export default function loadLocale(locale) {
-  return new Promise(resolve => {
-    if (hasIntlFor(locale)) {
-      return loadMessagesFor(locale).then(resolve.bind(this));
+import repeat from 'lodash/repeat';
+import partialRight from 'lodash/partialRight';
+import t from 'tcomb';
+
+export default function(config, loadMessagesFor) {
+  function longerStrings(data) {
+    if (!t.Num.is(config.localStringLengthMultiplier) || config.localStringLengthMultiplier === 1) {
```

Comment:
currently this `localStringLengthMultiplier` is not even allowed as config key if I'm not wrong. Since not very useful on the webseed project itself, I'd remove `longerStrings` (and `.then(longerStrings)` below) completely, to simplify even further